src/main/java/org/sonarsource/sonarlint/ls/SkippedPluginsNotifier.java

```diff
@@ -57,30 +59,50 @@ public void notifyOnceForSkippedPlugins(AnalysisResults analysisResults, Collect
           final var runtimeRequirement = (SkipReason.UnsatisfiedRuntimeRequirement) skipReason;
           final var title = String.format("SonarLint failed to analyze %s code", l.getLabel());
           if (runtimeRequirement.getRuntime() == SkipReason.UnsatisfiedRuntimeRequirement.RuntimeRequirement.JRE) {
```

Comment:
Nitpick: maybe `SkipReason.UnsatisfiedRuntimeRequirement.RuntimeRequirement` should be `static`-imported in this file.